driver/src/main/scala/core/nodeset.scala

```diff
@@ -667,6 +655,53 @@ final class ChannelFactory private[reactivemongo] (
     pipeline
   }
 
+  private def sslContext = {
+    import java.io.FileInputStream
+    import java.security.KeyStore
+    import javax.net.ssl.{ KeyManagerFactory, TrustManager }
+
+    val keyManagers = Option(System.getProperty("javax.net.ssl.keyStore")).map { path =>
+
+      val password = Option(System.getProperty("javax.net.ssl.keyStorePassword")).getOrElse("")
```

Comment:
Nice job :+1: I had a little look into the scripts but hadn't had the time to implement anything so thanks for getting this started. I've merged your commit in to my fork, today I'll have a go at the TODOs.